app/screens/navigation.ts

```diff
@@ -345,7 +345,6 @@ export function goToScreen(name: string, title: string, passProps = {}, options
             right: {enabled: false},
         },
         statusBar: {
-            backgroundColor: null,
```

Comment:
Why this change? I went based on https://wix.github.io/react-native-navigation/next/docs/style-statusBar/#how-to-keep-current-statusbar-color-when-displaying-screens

So if you explain this to me I'd be a happy man

Reply:
Maybe something is not working as expected, but that was the culprit of the white status bar.

I did not think so much through it: it seemed out of place and removing it solved the problem.

Maybe there is no "current status bar color", and somehow the logic fails and set it to white, but that would need deeper look into the react-native-navigation code.